### DISCUSSION TITLE
frame in outside, put into other windows

### DISCUSSION THREAD

**Discussion by coolH-alt** [fltk-rs/fltk-rs#1196] [Q&A]
i written a frame in win_1 outside, is there hava api to put the frame into win_1

use fltk::{app, enums::*, frame::Frame, group, menu::*, prelude::*, valuator::*, window::Window};
fn main() {
    let win_w = 1000;
    let win_h = 780;
    let app = app::App::default().with_scheme(app::AppScheme::Gtk);
    let bgc = Color::from_u32(0x696969);

    let mut win = Window::default()
        .with_size(win_w, win_h)
        .center_screen()
        .with_label("Test");

    let mut win_1 = Window::new(0, 0, win_w, 300, None);

    win_1.set_color(Color::from_u32(0xffffff));
    win_1.end();

    let mut tt = Frame::new(win_1.w() - 30, win_1.h() - 30, 30, 30, None);
    tt.set_frame(FrameType::FlatBox);
    tt.set_color(Color::Red);
    tt.size_of(&win_1);

    win.set_color(bgc);
    win.size_range(win_w, win_h, 0, 0);
    win.make_resizable(true);
    win.end();
    win.show();

    app.run().unwrap()
}

**Comment by MoAlyousef** [comment]
You can always use GroupExt::add:

`win_1.add(&tt);`
